src/test/java/tests/TurboMilestoneTest.java

```diff
@@ -34,6 +32,30 @@ public void turboMilestoneTest() {
     }
 
     @Test
+    public void getDueDateComparator_milestoneOrdering() {
```

Comment:
Change the test name to `getDueDateComparator_combinationOfDifferentMilestones_correctMilestoneOrdering`.